### PR TITLE
assignment 1

### PR DESCRIPTION
**Pull request by Rami-ElSkakni**

### CODE REVIEW

File: Styles.css [hunk 1]
```diff
@@ -1,366 +1,365 @@
...
-    color: white;
+  background-color: purple;
+  color: white;
 }
 
- h1 {
-    font-weight: 700;
-    text-align: center;
-    margin: 2em auto 0;
-    /* width: 670px; */
```
Comment (Rami-ElSkakni) [inline Styles.css]: remove comment here

File: index.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+            <img src="./images/clothing.jpeg" alt="clothes" class="project-img-3">
+            <div class="project-3-overlay project-3-overlay--primary ">
+              <div class="project-3-overlay__description">A website that displays all kinds of clothing-wear</div>
+             <a href="https://github.com/Rami-ElSkakni/Clothing-Front-End-Website"><button class="project-3-overlay__button">Visit Website</button></a> 
+           </div>
+          </div>   
+    </div>
+
+    <footer class="footer">
+        <img class="footer__img" src="./images/mf-logo-white.svg" alt="">
```
Comment (Chrishalaby) [inline index.html]: Dont make the class name like the tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+          </ul>
+      </div>
+    </div>
+        
+    <div class = "recent-work-container">
+      <h2 class="recent-work-container__h2">My Recent Work</h2>
+      <p class="recent-work-container__p">Here are a few design projects that I have worked on</p>
+
+      <div class="project-grid">
+          <div class="project-grid__1">
```
Comment (Chrishalaby) [inline index.html]: Way too many div tags that aren't required

File: index.html [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+          the front side of it since I prefer visuals and instantly seeing the changes
+          that I am doing. I'm currently doing an internship that is teaching me new things
+          and allowing me to sharpen my skills. After it, I would like to graduate from Computer
+          Science and start my career.
+        </p>
+      </div>
+    </section>
+
+    <div class="card-info">
+      <div class="card-info-1">
```
Comment (Chrishalaby) [inline index.html]: Dont use numbers for a classes card-info-1...

File: index.html [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+          <h2>Dev Tools:</h2>
+          <ul>
+            <li>Git</li>
+            <li>GitHub</li>
+            <li>Bootstrap</li>
+          </ul>
+      </div>
+    </div>
+        
+    <div class = "recent-work-container">
```
Comment (Chrishalaby) [inline index.html]: use sections instead of divs for your containers

File: index.html [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+          and allowing me to sharpen my skills. After it, I would like to graduate from Computer
+          Science and start my career.
+        </p>
+      </div>
+    </section>
+
+    <div class="card-info">
+      <div class="card-info-1">
+          <img src="./images/skills.svg" alt="">
+          <h1>Front-end Developer</h1>
```
Comment (Chrishalaby) [inline index.html]: dont over use headers. and dont use a header number more than once. use p instead and change the font in css

File: index.html [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+
+      <div class="project-grid">
+          <div class="project-grid__1">
+            <img src="./images/starbucks.jpg" alt="starbucks-cup-image" class="project-img-1">
+            <div class="project-1-overlay project-1-overlay--primary ">
+              <div class="project-1-overlay__description">This project was a clone of Starbucks' main page </div>
+              <a href="https://github.com/Rami-ElSkakni/Starbucks-FrontEnd-Clone"><button class="project-1-overlay__button">Visit Website</button></a>
+           </div>
+          </div>
+          <div class="project-grid__2">
```
Comment (Chrishalaby) [inline index.html]: a few BEM mistakes. not being consistent on the naming style

File: Styles.css [hunk 8]
```diff
@@ -0,0 +1,368 @@
...
+  border-radius: 24px;
+  border: none;
+  padding: 15px;
+  background-color: rgb(32, 48, 189);
+  color: white;
+  cursor: pointer;
+  margin-top: 9px;
+}
+
+.project-3-overlay__button {
```
Comment (Chrishalaby) [inline Styles.css]: code repetition that can be summed up

**Review by Chrishalaby — CHANGES_REQUESTED** [review]
Fix the commented parts

File: index.html (hunk 2 above)
Comment (Khousheish) [reply index.html]: i do agree that this shouldnt be the case but in this specific footer one i think it makes sense

File: index.html (hunk 6 above)
Comment (Khousheish) [reply index.html]: this shouldnt be the case
having only one header number only applies to h1
for all the lower numbers just be mindful if its a header use one of the h tags and use them in the proper sequence without skipping levels
only use p if its content

File: index.html [hunk 9]
```diff
@@ -0,0 +1,113 @@
...
+      href="http://fonts.cdnfonts.com/css/eurostile-extended"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+  </head>
+  <body>
+    <nav>
+      <img src="./images/mf-logo.svg" alt="" class="brand-img" />
+
+      <div class="nav-right">
```
Comment (Chrishalaby) [inline index.html]: div not needed

File: index.html [hunk 10]
```diff
@@ -0,0 +1,113 @@
...
+      </div>
+    </nav>
+
+    <main>
+      <h1>Front-end Developer</h1>
+      <p class="main-p">
+        I design and code beautifully simple things, and I love what I do
+      </p>
+      <img src="./images/mf-avatar.svg" alt="Graphic-profile-pic" class="img-avatar" />
+      <br />
```
Comment (Chrishalaby) [inline index.html]: br should never be used, find another way.

File: index.html [hunk 11]
```diff
@@ -0,0 +1,113 @@
...
+    <nav>
+      <img src="./images/mf-logo.svg" alt="" class="brand-img" />
+
+      <div class="nav-right">
+        <span class="nav-text">Mentorship</span>
+        <button class="nav-button">Say Hello</button>
+      </div>
+    </nav>
+
+    <main>
```
Comment (Chrishalaby) [inline index.html]: following bem rules, the block or (section) should always have the main name, with its subs followng that name

File: index.html [hunk 12]
```diff
@@ -0,0 +1,113 @@
...
+      <h1>Front-end Developer</h1>
+      <p class="main-p">
+        I design and code beautifully simple things, and I love what I do
+      </p>
+      <img src="./images/mf-avatar.svg" alt="Graphic-profile-pic" class="img-avatar" />
```
Comment (Chrishalaby) [inline index.html]: dont use the tag name in the class "img-..."

File: index.html [hunk 13]
```diff
@@ -0,0 +1,113 @@
...
+      <br />
+      <img src="./images/hero.svg" alt="Coding process" class="img-machine" />
+    </main>
+    <section class="personal-info-container">
+      <div class="text">
```
Comment (Chrishalaby) [inline index.html]: div not needed

File: index.html [hunk 14]
```diff
@@ -0,0 +1,113 @@
...
+          From when I was a teenager I was interested in Web development, especially 
+          the front side of it since I prefer visuals and instantly seeing the changes
+          that I am doing. I'm currently doing an internship that is teaching me new things
+          and allowing me to sharpen my skills. After it, I would like to graduate from Computer
+          Science and start my career.
+        </p>
+      </div>
+    </section>
+
+    <div class="card-info">
```
Comment (Chrishalaby) [inline index.html]: use section !!!

File: index.html [hunk 15]
```diff
@@ -0,0 +1,113 @@
...
+      <div class="card-info-skills">
```
Comment (Chrishalaby) [inline index.html]: another useless concatenation of div tags!

File: index.html [hunk 16]
```diff
@@ -0,0 +1,113 @@
...
+          <img src="./images/skills.svg" alt="">
```
Comment (Chrishalaby) [inline index.html]: add an alt

File: index.html [hunk 17]
```diff
@@ -0,0 +1,113 @@
...
+          <h2>Front-end Developer</h2>
+          <p class = "p-personal">I like to code things from scratch, and enjoy bringing ideas to life in the browser.</p>
```
Comment (Chrishalaby) [inline index.html]: wrong class name, follow the BEM!

File: index.html [hunk 18]
```diff
@@ -0,0 +1,113 @@
...
+            <li>Bootstrap</li>
+          </ul>
+      </div>
+    </div>
+        
+    <section class = "recent-work-container">
+      <h2 class="recent-work-container__h2">My Recent Work</h2>
+      <p class="recent-work-container__p">Here are a few design projects that I have worked on</p>
+      <div class="project-grid">
+          <div class="project-grid__item">
```
Comment (Chrishalaby) [inline index.html]: mentioned before, concatenation of div tags

File: Styles.css [hunk 19]
```diff
@@ -0,0 +1,335 @@
...
+  line-height: 1.7em;
+  font-weight: 100;
+  font-family: "Roboto", sans-serif;
+}
+
+.card-info {
+  display: grid;
+  grid-template-columns: repeat(1, 400px);
+  justify-content: center;
+  margin-top: -8em;
```
Comment (Chrishalaby) [inline Styles.css]: never use negative margins.

File: Styles.css [hunk 20]
```diff
@@ -0,0 +1,335 @@
...
+.card-info-skills .p-skills {
+  width: 310px;
+  color: black;
+  margin-top: 1.875em;
+  display: inline-block;
+  text-align: center;
+  font-weight: 300;
+}
+
+.card-info-skills ul {
```
Comment (Chrishalaby) [inline Styles.css]: dont use html tags in css

**Review by Chrishalaby — CHANGES_REQUESTED** [review]
Nothing new, same ideas that were previously mentioned.

File: index.html (hunk 9 above)
Comment (Rami-ElSkakni) [reply index.html]: don't I need this in order to group the button and the mentorship so that they stay near each other?

if there is another way, please tell me

File: index.html (hunk 15 above)
Comment (Rami-ElSkakni) [reply index.html]: I need this in case I wanted to add more cards into the grid, similar to the main website.

File: index.html (hunk 18 above)
Comment (Rami-ElSkakni) [reply index.html]: Also here I need the divs in order to have each card have grid applied to it. And the project-overlay div is for the hover animation.

I don't know if there is another way to do it other than using divs. If there is I would love to know :D

File: index.html (hunk 9 above)
Comment (Chrishalaby) [reply index.html]: you can apply the changes directly on the "nav-text" and from that point change whats needed for the "nav-buttton" to stay near each other.

Comment (Khousheish) [reply index.html]: or you can use flex
https://css-tricks.com/snippets/css/a-guide-to-flexbox/

File: index.html (hunk 18 above)
Comment (Khousheish) [reply index.html]: grid is a bit outdated consider usign flex as mentioned in another comment

File: Styles.css [hunk 21]
```diff
@@ -0,0 +1,342 @@
+* {
+  margin: 0;
+  padding: 0;
+  border-style: border-box;
+  font-family: "Roboto", sans-serif;
+}
+
+nav {
```
Comment (Khousheish) [inline Styles.css]: dont use html tags in css

File: Styles.css [hunk 22]
```diff
@@ -0,0 +1,342 @@
...
+  margin-top: 0.7em;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  height: 10vh;
+}
+
+.brand-img {
+  height: 52px;
```
Comment (Khousheish) [inline Styles.css]: cant these values be in em or rem?

File: Styles.css [hunk 23]
```diff
@@ -0,0 +1,342 @@
...
+
+.nav-text {
+  font-weight: 100;
+  margin-right: 1.4em;
+  font-size: smaller;
+}
+
+.nav-text:hover {
+  cursor: pointer;
+  color: purple;
```
Comment (Khousheish) [inline Styles.css]: dont use named colors use hexes and consider making css variables from these

File: Styles.css [hunk 24]
```diff
@@ -0,0 +1,342 @@
...
+}
+
+.nav-button {
+  font-size: 0.8em;
+  border: none;
```
Comment (Khousheish) [inline Styles.css]: you have 2 declarations of border here

File: Styles.css [hunk 25]
```diff
@@ -0,0 +1,342 @@
...
+h1 {
+  font-weight: 700;
+  text-align: center;
+  margin: 2em auto 0;
+  width: 720px;
+  display: inline-block;
+  font-size: 2.4em;
+}
+
+.Introduction {
```
Comment (Khousheish) [inline Styles.css]: use proper bem syntax

File: Styles.css [hunk 26]
```diff
@@ -0,0 +1,342 @@
...
+  line-height: 1.7em;
+  font-weight: 100;
+  font-family: "Roboto", sans-serif;
+}
+
+.card-info {
+  display: grid;
+  grid-template-columns: repeat(1, 400px);
+  justify-content: center;
+  /* margin-top: -8em; */
```
Comment (Khousheish) [inline Styles.css]: remove comments

File: Styles.css [hunk 27]
```diff
@@ -0,0 +1,342 @@
...
+  position: absolute;
```
Comment (Khousheish) [inline Styles.css]: dont use position absolute
when you change this to flex you wont need it anymore

File: Styles.css [hunk 28]
```diff
@@ -0,0 +1,342 @@
...
+.project-grid__img {
+  height: 200px;
+  width: 300px;
+  border-radius: 12px;
+  cursor: pointer;
+}
+
+.project-overlay {
+  color: white;
+  position: absolute;
```
Comment (Khousheish) [inline Styles.css:251]: same here

File: Styles.css [hunk 29]
```diff
@@ -0,0 +1,342 @@
...
+}
+
+.cinema-btn {
+  background-color: rgb(32, 48, 189);
+}
+
+.clothing-btn {
+  background-color: rgb(207, 52, 28);
+}
+
```
Comment (Khousheish) [inline Styles.css:318]: remove extra space

File: index.html [hunk 30]
```diff
@@ -0,0 +1,109 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="Styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
```
Comment (Khousheish) [inline index.html]: why is this included 2 times?

File: index.html [hunk 31]
```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="Styles.css" />
```
Comment (Khousheish) [inline index.html]: styles should be lowercase

File: index.html [hunk 32]
```diff
@@ -0,0 +1,109 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
```
Comment (Khousheish) [inline index.html]: remove all these and only use 1 font

File: index.html [hunk 33]
```diff
@@ -0,0 +1,109 @@
...
+              <span class="project-overlay__description">This project was a clone of Starbucks' main page </span>
+              <a href="https://github.com/Rami-ElSkakni/Starbucks-FrontEnd-Clone"><button class="project-overlay__button starbucks-btn">Visit Website</button></a>
+           </div>
+          </div>
+          <div class="project-grid__item">
+            <img src="./images/cinema.jpg" alt="cinema" class="project-grid__img">
+            <div class="project-overlay">
+              <span class="project-overlay__description">View and sort movies that you like based on a database</span>
+              <a href="https://github.com/Rami-ElSkakni/DataBase-Frontend-Project"><button class="project-overlay__button cinema-btn">Visit Website</button></a>
+           </div>
```
Comment (Khousheish) [inline index.html:86]: indentation seems wrong here

File: index.html [hunk 34]
```diff
@@ -0,0 +1,109 @@
...
+          </div>
+          <div class="project-grid__item">
```
Comment (Khousheish) [inline index.html]: leave an empty line between sibling blocks

**Review by Khousheish — CHANGES_REQUESTED** [review]

File: Styles.css (hunk 28 above)
Comment (Rami-ElSkakni) [reply Styles.css:251]: I need it here in order for the hover to work

File: Styles.css [hunk 35]
```diff
@@ -0,0 +1,326 @@
+* {
+  margin: 0;
+  padding: 0;
+  border-style: border-box;
+  font-family: "Roboto", sans-serif;
```
Comment (Chrishalaby) [inline Styles.css]: Place the Font family in a variable

File: Styles.css [hunk 36]
```diff
@@ -0,0 +1,326 @@
...
+  height: 3.25em;
+  width: 3em;
+  cursor: pointer;
+  margin-left: 10em;
+}
+
+.navbar__text {
+  font-weight: 100;
+  margin-right: 1.4em;
+  font-size: smaller;
```
Comment (Chrishalaby) [inline Styles.css]: Be consistent with the font size units, used smaller here, and a few lines down used 0.8em.

File: Styles.css [hunk 37]
```diff
@@ -0,0 +1,326 @@
...
+.navbar__text:hover {
+  cursor: pointer;
+  color: var(--purple-color);
+}
+
+.navbar__button {
+  font-size: 0.8em;
+  padding: 0.6rem;
+  border-radius: 36px;
+  background-color: #FFFFFF;
```
Comment (Chrishalaby) [inline Styles.css]: Use variables for colors hexes

File: Styles.css [hunk 38]
```diff
@@ -0,0 +1,326 @@
...
+.navbar__button:hover {
+  background-color: var(--purple-color);
+  color: #FFFFFF;
+}
+
+.introduction__header {
+  font-weight: 700;
+  text-align: center;
+  margin: 2em auto 0;
+  display: inline-block;
```
Comment (Chrishalaby) [inline Styles.css]: use flex

File: Styles.css [hunk 39]
```diff
@@ -0,0 +1,326 @@
...
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.introduction__p {
+  text-align: center;
+  display: block;
```
Comment (Chrishalaby) [inline Styles.css]: use flex

Comment (Chrishalaby) [reply Styles.css]: should be applied everywhere

File: Styles.css [hunk 40]
```diff
@@ -0,0 +1,326 @@
...
+  display: inline-block;
+}
+
+.introduction__heroMachine {
+  width: 32.5em;
+  margin-top: 3.7em;
+}
+
+.personal-info-container {
+  background-color: #7510f7;
```
Comment (Chrishalaby) [inline Styles.css]: move to variable...

File: Styles.css [hunk 41]
```diff
@@ -0,0 +1,326 @@
...
+  display: flex;
+  height: 31.25em;
```
Comment (Chrishalaby) [inline Styles.css]: Height should be removed, let the element take the width needed by itself, use only margin and padding

File: Styles.css [hunk 42]
```diff
@@ -0,0 +1,326 @@
...
+  align-items: center;
+  flex-direction: column;
+  position: relative;
+}
+
+.personal-info-container__header {
+  text-align: center;
+  margin-top: 6.25em;
+  color: white;
```
Comment (Chrishalaby) [inline Styles.css]: use hex.. apply it everywhere

File: Styles.css [hunk 43]
```diff
@@ -0,0 +1,326 @@
...
+  font-family: "Roboto", sans-serif;
```
Comment (Chrishalaby) [inline Styles.css]: remove font family, should defined globally at the top

File: Styles.css [hunk 44]
```diff
@@ -0,0 +1,326 @@
...
+  align-items: center;
+}
+
+ .card-info__skills__svg {
+  height: 50px;
+  width: 48px;
+  padding-top: 3.125em;
+}
+
+.card-info__skills h2 {
```
Comment (Chrishalaby) [inline Styles.css]: dont use html tags...

File: index.html [hunk 45]
```diff
@@ -0,0 +1,92 @@
...
+  </head>
+  <body>
+    <nav class="navbar">
+      <img src="./images/mf-logo.svg" alt="main logo" class="navbar__logo" />
+      <span class="navbar__text">Mentorship</span>
+      <button class="navbar__button">Say Hello</button>
+    </nav>
+    <main class="introduction">
+      <h1 class="introduction__header">Front-end Developer</h1>
+      <p class="introduction__p">
```
Comment (Chrishalaby) [inline index.html]: use smtg other than html tags for the class name.

File: index.html [hunk 46]
```diff
@@ -0,0 +1,92 @@
...
+    <section class="personal-info-container">
+        <h2 class="personal-info-container__header">Hi, I'm Rami. Nice to meet you</h2>
+        <p class ="personal-info-container__introduction">
+          From when I was a teenager I was interested in Web development, especially 
+          the front side of it since I prefer visuals and instantly seeing the changes
+          that I am doing. I'm currently doing an internship that is teaching me new things
+          and allowing me to sharpen my skills. After it, I would like to graduate from Computer
+          Science and start my career.
+        </p>
+        <section class="card-info">
```
Comment (Chrishalaby) [inline index.html]: Never use a section inside another Section.

File: index.html [hunk 47]
```diff
@@ -0,0 +1,92 @@
...
+          <div class="card-info__skills">
```
Comment (Chrishalaby) [inline index.html]: div not needed here

File: index.html [hunk 48]
```diff
@@ -0,0 +1,92 @@
...
+            </ul>
+          </div>
+        </section>
+    </section>
+
+    <section class = "recent-work-container">
+      <h2 class="recent-work-container__h2">My Recent Work</h2>
+      <p class="recent-work-container__p">Here are a few design projects that I have worked on</p>
+      <div class="project-flex">
+          <div class="project-flex__item">
```
Comment (Chrishalaby) [inline index.html]: no need for concatenated divs.

File: index.html [hunk 49]
```diff
@@ -0,0 +1,92 @@
...
+            <img src="./images/starbucks.jpg" alt="starbucks-cup-image" class="project-flex__img">
+            <div class="project-overlay">
+              <span class="project-overlay__description">This project was a clone of Starbucks' main page </span>
+              <a href="https://github.com/Rami-ElSkakni/Starbucks-FrontEnd-Clone"><button class="project-overlay__button starbucks-btn">Visit Website</button></a>
+           </div>
+
+          </div>
+          <div class="project-flex__item">
```
Comment (Chrishalaby) [inline index.html:79]: same idea as before... no need for concatenated divs

**Review by Chrishalaby — CHANGES_REQUESTED** [review]
other than the mentioned comments, pls recheck your BEM namings. be consistent with it

File: index.html [hunk 50]
```diff
@@ -0,0 +1,92 @@
...
+            <ul class="card-info__skills__list">
+              <li class="card-info__skills__list__item">Git</li>
+              <li class="card-info__skills__list__item">GitHub</li>
+              <li class="card-info__skills__list__item">Bootstrap</li>
+            </ul>
+          </div>
+        </div>
+    </section>
+
+    <section class = "recent-work-container">
```
Comment (Khousheish) [inline index.html]: remove spaces

**Review by Khousheish — CHANGES_REQUESTED** [review]

**Review by Khousheish — APPROVED** [review]
well done! looks good @Chrishalaby do a final check from your side and lets move on!

File: index.html [hunk 51]
```diff
@@ -0,0 +1,113 @@
...
+
+      <p class ="personal-info-container__introduction">
+        From when I was a teenager I was interested in Web development, especially 
+        the front side of it since I prefer visuals and instantly seeing the changes
+        that I am doing. I'm currently doing an internship that is teaching me new things
+        and allowing me to sharpen my skills. After it, I would like to graduate from Computer
+        Science and start my career.
+      </p>
+
+      <div class="card-info">
```
Comment (Chrishalaby) [inline index.html:40]: remove concatenated div tags. no need for divs here too

File: index.html [hunk 52]
```diff
@@ -0,0 +1,113 @@
...
+      </div>
+    </section>
+
+    <section class="recent-work-container">
+      <h2 class="recent-work-container__header">My Recent Work</h2>
+
+      <p class="recent-work-container__description">Here are a few design projects that I have worked on</p>
+
+      <div class="project-flex">
+        <div class="project-flex__item">
```
Comment (Chrishalaby) [inline index.html:69]: same idea as above

File: index.html [hunk 53]
```diff
@@ -0,0 +1,113 @@
...
+
+              <a href="https://github.com/Rami-ElSkakni/Clothing-Front-End-Website"><button class="project-overlay__button project-overlay__button--clothing">Visit Website</button></a> 
+           </div>
+          </div>   
+    </section>
+
+    <footer class="footer">
+      <img class="footer__img" src="./images/mf-logo-white.svg" alt="footer logo">
+
+      <div class="footer__description">Living, learning, & leveling up one day at a time.</div>
```
Comment (Chrishalaby) [inline index.html:133]: use a p tag instead of div here

File: index.html [hunk 54]
```diff
@@ -0,0 +1,113 @@
...
+
+      <div class="icons-container">
+        <span class="icons-container__list"><i class="fa-solid fa-envelope fa-icon"></i></span>
+        <span class="icons-container__list"><i class="fa-brands fa-linkedin-in fa-icon"></i></span>
+        <span class="icons-container__list"><i class="fa-brands fa-github fa-icon"></i></span>
+      </div>
+
+      <div class="footer__doneby">Built By me</div>
```
Comment (Chrishalaby) [inline index.html:141]: same as above here

File: Styles.css [hunk 55]
```diff
@@ -0,0 +1,339 @@
...
+  margin-top: calc(var(--margin-spacer) * 3);
+}
+
+.project-flex__item {
+  position: relative;
+  margin-right: calc(var(--margin-spacer) * 1.25);
+}
+
+.project-flex__img {
+  height: 200px;
```
Comment (Chrishalaby) [inline Styles.css:243]: use rem or em. applied to everywhere

**Review by Chrishalaby — CHANGES_REQUESTED** [review]
Overall well done, try to go over 1 more time on the BEM and do some minor changes to be consistent with it.

File: Styles.css [hunk 56]
```diff
@@ -282,6 +292,87 @@
...
 }
 
+.testimonials__carousel {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  margin-top: calc(var(--margin-spacer) * 3);
+  font-size: calc(var(--font-size) * 1);
+  display: none;
```
Comment (Khousheish) [inline Styles.css:302]: why is this here?

File: Styles.css [hunk 57]
```diff
@@ -282,6 +292,87 @@
...
+}
+
+.testimonials__carousel__description {
+  display: flex;
+  justify-content: center;
+  max-width: 30em;
+  text-align: center;
+  font-weight: 300;
+  font-size: calc(var(--font-size) * 1.25);
+  margin-left: calc(var(--margin-spacer) * 3);
```
Comment (Khousheish) [inline Styles.css]: this can be written in a better way without having to write both left and right
you can do margin: 0 calc(var(--margin-spacer) * 3);

File: index.html [hunk 58]
```diff
@@ -96,6 +96,37 @@ <h2 class="recent-work-container__header">My Recent Work</h2>
...
+
+        <p class="testimonials__carousel__description">“Matt was a real pleasure to work with and we look forward to working with him again.
+          He's definitely the kind of designer you can trust with a project from start to finish.”</p>
+      
+        <div class="testimonials__carousel__name">Pascal Trembly</div>
+
+        <div class="testimonials__carousel__signature">Creative lead, Good Kind</div>
+      </div>
+      <div class="testimonials__carousel">
+        <img src="./images/alvin.png" alt="picture of my employer" class="testimonials__carousel__picture">
```
Comment (Khousheish) [inline index.html:114]: this is an ok solution but if you want to go the extra step why not iterate over and array of objects containing the config for the carousel? so its more easily upgradable for the future

**Review by Khousheish — APPROVED** [review]

File: Styles.css (hunk 56 above)
Comment (Rami-ElSkakni) [reply Styles.css:302]: this is used in order to toggle the testimonial that is shown in the carousel. It starts off with none displayed then I give it another class that changes the display to show it once the button is clicked

**Review by Chrishalaby — APPROVED** [review]
